app/src/main/java/com/android/streetworkapp/ui/train/AnimatedCounter.kt

```diff
@@ -42,7 +43,11 @@ fun AnimatedCounter(
             }
           },
           label = "") { c ->
-            Text(text = c.toString(), style = style, softWrap = false)
+            Text(
+                text = c.toString(),
```

Comment:
Refactor: Perhaps a more significant name than "c" could be used

Reply:
Done